system-tests/playwright/playwright.todolistapp/model/toDoList.page.js

```diff
@@ -0,0 +1,78 @@
+const BasePage = require('./Base.page');
+const {test} = require("@playwright/test");
+class ToDoListPage extends BasePage {
+    constructor(page){
+        super(page);
+
+        //selectors
+        this.toDoInput=".new-todo";
+        this.selectBtn=".toggle"
+
+
+        // locators
+       this.toDoList= page.locator(".todo-list li");
+       this.toggleAllBtn=page.locator("#toggle-all");
+       this.active=page.getByText(('Active'),{exact: true})
+        this.complete=page.getByText(('Completed'),{exact: true})
+        this.deleteBtn=page.locator(".destroy");
+
+
+
+    }
+
+    async navigate(){
+        await super.navigate('#/');
+    }
+    async createToDo(input){
+      await test.step('Filling the task input',async ()=>{
+          await this.page.fill(this.toDoInput,input);
+      })
+        await test.step('Press \'Enter\' to add the new task',async ()=>{
```

Comment:
formatting is all over the place, please install and apply `typescript-eslint` or whatever is most useful for that